src/api/java/baritone/api/Settings.java

```diff
@@ -752,6 +752,12 @@
      */
     public final Setting<Boolean> layerOrder = new Setting<>(false);
 
+    /**
+     * Start building the schematic at a specific layer.
+     * Can help on larger builds when schematic wants to break things its already built
+     */
+    public final Setting<Integer> startAtLayer = new Setting<>(1);
```

Comment:
Shouldn't this default to 0 if the last default was 0??? Small brain lejiurv for merging

Reply:
it's faster to merge then fix yourself then get the author to fix a single character https://github.com/cabaletta/baritone/commit/b187d860741c7e5b6cb6aab64a5b80dcdadec9a9

Reply:
I missed that :p

Reply:
Also you can edit other people's branches if they made a pull request to your repo lmao

Reply:
how rude